test/test_inventory.py

```diff
@@ -0,0 +1,16 @@
+from itertools import product
+from inventory.upcloud import list_servers, get_server
+
+class TestInventory(object):
+    def test_list_servers(self, manager):
+        IPvs_to_test = ['IPv4', 'IPv6']
+        possible_configs = list(product([True, False], repeat=2))
+
+        for IP_v in IPvs_to_test:
+            for config in possible_configs:
+                list_servers(manager, config[0], config[1], IP_v)
```

Comment:
Please add some assertions to check the `list_servers` returns the expected result